source/slang/slang-ir-insts.lua

```diff
@@ -1473,13 +1473,14 @@ local insts = {
 					struct_name = "RequireFullQuadsDecoration",
 				},
 			},
-			{ TempCallArgVar = { struct_name = "TempCallArgVarDecoration" } },
+			{ TempCallArgVar = { struct_name = "TempCallArgImmutableVarDecoration" } },
```

Comment:
I greatly appreciate the addition of `Immutable` to this name, since it may help ensure that in subsequent edits we don't go and start using the same opcode for other, incompatible, uses.

It would be nice to see the same thing done to the mnemonic on the left side there, since that is what determines the `kIROp_*` enumerant.